src/components/EReceiptThumbnail.tsx

```diff
@@ -1,31 +1,29 @@
-import PropTypes from 'prop-types';
 import React, {useMemo, useState} from 'react';
-import {View} from 'react-native';
-import {withOnyx} from 'react-native-onyx';
+import {LayoutChangeEvent, View} from 'react-native';
+import {OnyxEntry, withOnyx} from 'react-native-onyx';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as ReportUtils from '@libs/ReportUtils';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {Transaction} from '@src/types/onyx';
 import Icon from './Icon';
 import * as eReceiptBGs from './Icon/EReceiptBGs';
 import * as Expensicons from './Icon/Expensicons';
 import * as MCCIcons from './Icon/MCCIcons';
 import Image from './Image';
-import transactionPropTypes from './transactionPropTypes';
 
-const propTypes = {
-    /* TransactionID of the transaction this EReceipt corresponds to */
-    // eslint-disable-next-line react/no-unused-prop-types
-    transactionID: PropTypes.string.isRequired,
-
-    /* Onyx Props */
-    transaction: transactionPropTypes,
+type EReceiptThumbnailOnyxProps = {
+    transaction: OnyxEntry<Transaction>;
 };
 
-const defaultProps = {
-    transaction: {},
+type EReceiptThumbnailProps = EReceiptThumbnailOnyxProps & {
+    /** TransactionID of the transaction this EReceipt corresponds to
+     * It's used by withOnyx HOC
+     */
```

Comment:
```suggestion
    /** TransactionID used by withOnyx of the transaction this EReceipt corresponds to */
```

Prop comments are preferred to be single lines.

Reply:
Done!